Aboutme.md

```diff
@@ -0,0 +1,19 @@
+### :open_file_folder: What have I learnt?
+* The different principles of Software Development.
+* How to compile a Java
```

Comment:
Should say - how to compile a Java program

Reply:
Thank you. I will make the changes